recipe/meta.yaml

```diff
@@ -356,8 +376,10 @@ outputs:
         # https://github.com/conda-forge/tensorflow-feedstock/issues/162
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
       # host requirements to pick up run_exports
+        - libabseil
+        - grpc-cpp
```

Comment:
This should come before (not after) the comment. And again: why is it a build dep?
```suggestion
        - libabseil
        - grpc-cpp
      # host requirements to pick up run_exports
```

Reply:
They would be pulled in as host-deps anyways but I don't know why it would be beneficial to have them in `build` as long as you aren't cross-compiling.